src/pkg/packaging/dir.proj

```diff
@@ -81,14 +81,16 @@
       <templateSvg>$(ProjectDir)resources/images/version_badge.svg</templateSvg>
     </PropertyGroup>
 
+    <MakeDir  Condition="!Exists('$(BaseOutputRootPath)')" Directories="$(BaseOutputRootPath)" />
+    
     <WriteLinesToFile
       File="$(OutputVersionBadge)"
       Lines="$([System.IO.File]::ReadAllText('$(templateSvg)').Replace('ver_number', '$(ProductVersion)'))"
       Overwrite="true"
                       />
   </Target>
   
-  <Target Name="GenerateCompressedFiles" DependsOnTargets="InitPackage;GenerateZip;GenerateTarBall" />
+  <Target Name="GenerateCompressedFiles" DependsOnTargets="InitPackage;GenerateZip;GenerateTarBall" Condition="'$(UsePrebuiltPortableBinaries)' == 'false'"/>
```

Comment:
Unless I am missing something, post your change, the portable build that you are enabling (the binaries of which are being consumed by distro specific installers) is the only portable build that is going to be done (otherwise, we will have the problem of doing two build of portable binaries, where one set will be going into installers and another going into tar files).

Tar files need to be generated for portable build as well and by whichever is going to be the portable build step after your change (seems to me that is your change). Also, distro specific builds will be likely going away.